### ISSUE TITLE
Unclear issue when trying to build the site

### ISSUE THREAD

**Issue by fredericsimard** [mkdocs/mkdocs#3014]
Hi, on two different machines (one macOS, the other Ubuntu with a fresh install of mkdocs), when I try to build the site, I get these very unclear error messages. Would anyone have an idea if the issue is with the mkdocs files or python itself?

Running python 3.10 on the Mac, 3.8 on Ubuntu. Latest version of mkdocs and materials installed. The YAML file was validated, but it is encoded in ASCII and multiple attempts to convert it to UTF8 have mysteriously failed.

It also fails when compiling with GitHub Pages, which is why I tried to debug this on my laptop. There were a few changes made to the code by removing a plug-in I believe, but even if I download old commits that used to build and publish just fine do not work on my computer or on Ubuntu.

I'm really at a loss here.

```
mkdocs build
INFO     -  DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
              File "/usr/lib/python3/dist-packages/apport/report.py", line 13, in <module>
                import fnmatch, glob, traceback, errno, sys, atexit, locale, imp, stat
              File "/usr/lib/python3.8/imp.py", line 31, in <module>
                warnings.warn("the imp module is deprecated in favour of importlib; "
Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/__main__.py", line 247, in build_command
    cfg = config.load_config(**kwargs)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/base.py", line 363, in load_config
    errors, warnings = cfg.validate()
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/base.py", line 228, in validate
    run_failed, run_warnings = self._validate()
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/base.py", line 186, in _validate
    self[key] = config_option.validate(value)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/config_options.py", line 146, in validate
    return self.run_validation(value)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/config_options.py", line 937, in run_validation
    name, plugin = self.load_plugin_with_namespace(name, cfg)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/config_options.py", line 974, in load_plugin_with_namespace
    return (name, self.load_plugin(name, config))
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/config_options.py", line 1002, in load_plugin
    self.warnings.extend(f"Plugin '{name}' value: '{x}'. Warning: {y}" for x, y in warnings)
  File "/usr/local/lib/python3.8/dist-packages/mkdocs/config/config_options.py", line 1002, in <genexpr>
    self.warnings.extend(f"Plugin '{name}' value: '{x}'. Warning: {y}" for x, y in warnings)
ValueError: too many values to unpack (expected 2)
```

**Comment by oprypin** [comment]
Uh oh, this is likely a regression in MkDocs 1.4.1

For now please try to install `mkdocs==1.4.0` and let me know if that works

**Comment by oprypin** [comment]
It would be really helpful to be able to reproduce this issue myself and see which plugin is causing this. Would you be able to share your mkdocs.yml ?

**Comment by fredericsimard** [comment]
Yes of course, here it is:

```
site_name: General Bikeshare Feed Specification
site_url: https://gbfs.mobilitydata.org
repo_url: https://github.com/nabsa/gbfs

theme:
  logo_light_mode: assets/venn.svg
  logo_dark_mode: assets/library-white.svg
  palette:
    - media: "(prefers-color-scheme: light)"
      scheme: default
      primary: blue
      accent: blue
  name: material
  custom_dir: overrides
  favicon: assets/venn-black.svg
  font:
    text: SF Pro
    code: SF Mono
  features:
     - navigation.tabs
     - navigation.indexes
     - navigation.top
     - toc.integrate
  custom_dir: theme_overrides
extra_css:
  - assets/stylesheets/extra.css
extra:
  analytics:
    provider: google
    property: G-N7RXJMH84M
  social:
    - icon: fontawesome/brands/github
      link: https://github.com/MobilityData
    - icon: fontawesome/brands/linkedin
      link: https://www.linkedin.com/company/mobilitydata
    - icon: fontawesome/brands/slack
      link: https://bit.ly/mobilitydata-slack
markdown_extensions:
  - attr_list
  - md_in_html
  - admonition
  - toc:
      permalink: true
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
  - pymdownx.tasklist:
      custom_checkbox: true
  - pymdownx.superfences

plugins:
  - search
  - redirects:
      redirect_maps:
        'home.md': 'index.md'
        'major-change-proposals.md': 'specification/change-proposals.md'
        'tools-services.md': 'toolbox/index.md'
        'faq.md': 'learn/faq.md'
        'about.md': 'participate.md'
  - mkdocs-simple-hooks:
      hooks:
        on_config: "hooks.on_config:deploy_hooks"

nav:
- Home: index.md
- Specification:
  - specification/index.md
  - Current Version: specification/reference.md
  - Change Proposals: specification/change-proposals.md
  - Governance Process: specification/process.md
- Toolbox: 
  - toolbox/index.md
  - Community Tools and Resources: toolbox/resources.md
  - Glossary: toolbox/glossary.md
- Learn:
  - learn/index.md
  - White Papers: 
    - learn/white-papers/index.md
    - Data Policy: learn/white-papers/data-policy.md
    - Data Policy Europe: learn/white-papers/data-policy-europe.md
  - FAQ: learn/faq.md
- Participate: participate.md
```

**Comment by fredericsimard** [comment]
Holy molly, regressing to 1.4.0 has fixed the issue :-O

**Comment by fredericsimard** [comment]
By the way this is public and open source documentation, you can recreate the entire site yourself from this repo: https://github.com/MobilityData/gbfs.mobilitydata.org

**Comment by oprypin** [comment]
BTW your site's config simply had an issue

`WARNING  -  Config value: 'plugins'. Warning: Plugin 'mkdocs-simple-hooks'. Warning: Module 'hooks.on_config' doesn't have attribute 'deploy_hooks'.`

It's just that trying to print that issue caused a fatal problem...

If you happen to not trigger that issue anymore, you could switch to latest MkDocs again.

I have published a fix. Not sure when it will be released.

**Comment by fredericsimard** [comment]
Yes, after I posted this issue, I was able to reach the person who used to take care of this repo and they assured me I could safely remove that plug-in as it was unnecessary. I'll wait for the fix to come out before I try 1.4.1 again. Thanks
